apps/antalmanac/src/components/Calendar/CalendarRoot.tsx

```diff
@@ -159,6 +160,9 @@ export default function ScheduleCalendar(props: ScheduleCalendarProps) {
     const calendarStyling = isMobile ? { height: `calc(100% - 55px)` } : { height: `calc(100vh - 104px)` };
     const calendarTimeFormat = isMilitaryTime ? 'HH:mm' : 'h:mm A';
     const calendarGutterTimeFormat = isMilitaryTime ? 'HH:mm' : 'h A';
+    const defaultFinals = getDefaultFinalsStartDate();
+    const finalsDateFormat = defaultFinals ? 'ddd MM/DD' : 'ddd';
+    const date = showFinalsSchedule ? defaultFinals : new Date(2018, 0, 1);
```

Comment:
```suggestion

    const calendarStyling = isMobile ? { height: `calc(100% - 55px)` } : { height: `calc(100vh - 104px)` };
    
    const calendarTimeFormat = isMilitaryTime ? 'HH:mm' : 'h:mm A';
    const calendarGutterTimeFormat = isMilitaryTime ? 'HH:mm' : 'h A';
    
    const defaultFinals = getDefaultFinalsStartDate();
    const finalsDateFormat = defaultFinals ? 'ddd MM/DD' : 'ddd';
    const date = showFinalsSchedule ? defaultFinals : new Date(2018, 0, 1);
```

Honestly, this is a total nitpick (not your fault/not related to this PR), and could be a code smell, but I'd like to break up some of these values so it's more immediately clear what is related to what.